cdk-v2/Vagrantfile

```diff
@@ -9,6 +9,10 @@
 # * Have valid RH employee subscription account
 # * Have 'vagrant-adbinfo' plugin installed
 
+Vagrant.require_version ">= 1.7.4"
```

Comment:
installing centos 1.74 package from official vagrant site works but then requires me to reinstall all plugins
```
Vagrant failed to initialize at a very early stage:

The plugins failed to load properly. The error message given is
shown below.

Unable to find binary extension, was hitimes installed correctly?
```